README.md

```diff
@@ -665,6 +665,22 @@ bash assets/create.sh
 cargo install --path . --locked --force
 ```
 
+## Contributors
+
+Please don't forget to update `CHANGELOG.md` and describe your changes, this makes the release of a new `bat` version much easier.
+
+Each entry should follow the next format:
+- Short description of the change (one line)
+- Number of the pull request
+- Github's username
+
+See the next examples taken from `CHANGELOG.md`
```

Comment:
```suggestion
See the following examples taken from `CHANGELOG.md`
```
Note, I'm not sure we really need to call out some real examples from the changelog here, the instructions we post on PRs is (IMO) quite concise and clear:

> Could you please add an entry to the "unreleased" section in CHANGELOG.md? The format for entries is:
> `- Description what has been changed, see #123 (@user)`
> (where `#123` links to the bug ticket and/or the PR and user is your username.)